packages/core/modules/viewport-units/ViewportUnits.ts

```diff
@@ -0,0 +1,26 @@
+import { defineModule, onInit, ref } from 'ovee.js';
+
+export const ViewportUnits = defineModule(({ app }) => {
+	const vh = ref(0);
+	const vw = ref(0);
+
+	onInit(() => {
+		updateUnits();
+
+		app.$on('resize', updateUnits, { target: window });
+	});
+
+	function updateUnits() {
+		vh.value = document.documentElement.clientHeight / 100;
+		vw.value = window.innerWidth / 100;
+
+		document.documentElement.style.setProperty('--vh', `${vh.value}px`);
+		document.documentElement.style.setProperty('--vw', `${vw.value}px`);
+	}
+
+	return {
+		vh,
+		vw,
+		updateUnits,
+	};
+});
```

Comment:
Proponuję dodać globalny state dostępny w $app z tymi wartościami i dla uproszczenia dostępu, composable który także je wyeksponuje. Może i trochę edge case, ale dałoby to nam dostęp do tych wartości na poziomie kodu, konsystentnych z CSSem bez zastanawiania się z jakiego property w `window/document` brać wartość.